src/services/hospitalInfo/hospitalInfoService.ts

```diff
@@ -1,70 +1,73 @@
-import { airtableService } from '../../mapping/airtableService';
-import { HospitalInfo } from '../../models/hospitalInfo';
+import { airtableService } from "../../mapping/airtableService";
+import { HospitalInfo } from "../../models/hospitalInfo";
 
 import thumbnailData from "../../../test/thumbnailData.json";
 
 const extractUrls = (attachments: any) => {
-    return attachments && attachments.length > 0 ? attachments.map((att: any) => att.url) : [];
+  return attachments && attachments.length > 0
```

Comment:
minor: You don't need the length check.

Reply:
I modified it like this as your feedback.
`const extractUrls = (attachments: any) => {
  return attachments ? attachments.map((att: any) => att.url) : [];
};`

Reply:
minor: to be even more precise the type definition should be  `attachments: any[]`.